SYCL/XPTI/basic_event_collection.cpp

```diff
@@ -0,0 +1,125 @@
+// REQUIRES: xptifw, opencl
+// RUN: %clangxx %s -DXPTI_COLLECTOR -L%xptifw_lib -I%xptifw_includes -shared -fPIC -std=c++17 -o %t_collector.dll -lxptifw
```

Comment:
the command line will not work is clang-cl is used (-std=c++17 and -fPIC will not work).

There ar substitutions for such a case: https://github.com/intel/llvm-test-suite/blob/intel/SYCL/lit.cfg.py#L139